libtelemetry/src/main/java/com/mapbox/android/telemetry/CertificateBlacklist.java

```diff
@@ -80,7 +80,7 @@
         try {
           blacklist = obtainBlacklistContents(file);
           blacklist.remove(BLACKLIST_HEAD);
-        } catch (IOException exception) {
+        } catch (IOException | IndexOutOfBoundsException exception) {
```

Comment:
@electrostat can you please look into why `blacklist.remove(BLACKLIST_HEAD)` throws `IOBE`?